bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -92,14 +86,12 @@ class Presentation extends PureComponent {
       ignorePresentationRestoring: true,
     };
 
-    const PAN_ZOOM_INTERVAL = window.meetingClientSettings.public.presentation.panZoomInterval || 200;
-
-    this.currentPresentationToastId = 'currentPresentationToastId';
+    const PAN_ZOOM_INTERV = window.meetingClientSettings.public.presentation.panZoomInterval || 200;
```

Comment:
Possible typo! I don't know if that's intentional, but `PAN_ZOOM_INTERVAL` is written `PAN_ZOOM_INTERV`.
If that's a typo, there is a reference of this variable in line 94

Reply:
It was necessary because the line was too long 😅